Extensions/xinbox.js

```diff
@@ -683,9 +683,9 @@ XKit.extensions.xinbox = new Object({
 
 	resize_text_area: function(post_id) {
 
-		if ($("#ask_answer_field_" + (+post_id) + "_tbl").length > 0) {
```

Comment:
oh so thats how i missed it. oooh i hate that

Reply:
its okay, i don't think this code works anymore anyway